hedera-node/src/main/java/com/hedera/services/txns/contract/ContractUpdateTransitionLogic.java

```diff
@@ -119,6 +158,7 @@ public ResponseCodeEnum validate(TransactionBody contractUpdateTxn) {
 			}
 		}
 
+		//TODO: The op.getMemo() is deprecated. What method should be used when memo is not wrapped?
```

Comment:
We should always use the `memoWrapper`. Memo was deprecated because you cannot differentiate between:
`clearing out the memo field to empty string` and `I do not want to update the memo field so I leave the memo as default property in the update TX`.

The default property for `string` is `""` which is the same as if you would want to "remove" the memo field trough update